<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>pythonUtilsConfigWidget</class>
    <widget class="QWidget" name="djangoConfigWidget">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
                <layout class="QVBoxLayout" name="verticalLayout">
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout">
                            <item>
                                <widget class="QLabel" name="label">
                                    <property name="text">
                                        <string>Django tags to close when call the "Close Template tag" action</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLineEdit" name="templateTagsClose">
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout">
                            <item>
                                <widget class="QLabel" name="label">
                                    <property name="text">
                                        <string>URL template file</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QTextEdit" name="templateTextURLs">
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout">
                            <item>
                                <widget class="QLabel" name="label">
                                    <property name="text">
                                        <string>View import snippet</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QTextEdit" name="templateTextViews">
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout">
                            <item>
                                <widget class="QLabel" name="label">
                                    <property name="text">
                                        <string>Pattern Model Form</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QTextEdit" name="patternModelForm">
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout">
                            <item>
                                <widget class="QLabel" name="label">
                                    <property name="text">
                                        <string>Pattern Model</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QTextEdit" name="patternModel">
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <spacer name="verticalSpacer">
                            <property name="orientation">
                                <enum>Qt::Vertical</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>20</width>
                                    <height>40</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
